content/post/2.5.1.md

```diff
@@ -2,7 +2,7 @@
 title: Release 2.5.1 is available
 date: 2023-05-05
 type: release
-linked: true
+linked: false
```

Comment:
@adoroszlai , thanks a lot for working on this!  Let's keep 2.5.1?  Someone may be still using branch-2.